ui/components/MeshConfigComponent.js

```diff
@@ -257,6 +257,49 @@ function MesherySettingsNew({ classes, enqueueSnackbar, closeSnackbar, updatePro
     });
   }
 
+  const handleConfigSnackbars = ctxs => {
+    updateProgress({ showProgress : false });
+
+    for (let ctx of ctxs.inserted_contexts) {
+      const msg = `Cluster ${ctx.name} at ${ctx.server} added successfully`
+      enqueueSnackbar(msg, {
+        variant : "success",
+        action : (key) => (
+          <IconButton key="close" aria-label="Close" color="inherit" onClick={() => closeSnackbar(key)}>
+            <CloseIcon />
+          </IconButton>
+        ),
+        autoHideDuration : 2000,
+      });
+    }
+
+    for (var ctx of ctxs.updated_contexts) {
+      const msg = `Cluster ${ctx.name} at ${ctx.server} already exists`
```

Comment:
Please consider this as an info, not a warning.

Reply:
Alright :+1: